src/Moontoast/Math/BigNumber.php

```diff
@@ -647,10 +647,11 @@ public static function convertFromBase10($number, $toBase)
         $outNumber = '';
         
         while (bccomp($number, $toBase) >= 0) {
-            $quotient = bcdiv($number, $toBase, 0);
-            $remainder = bcsub($number, bcmul($quotient, $toBase, 0), 0);
+            list($number, $remainder) = array(
+                bcdiv($number, $toBase, 0),
+                bcmod($number, $toBase)
+            );
```

Comment:
You are probably right. The array is not necessary to wrap, we can simply change the order:

```
$remainder = bcmod($number, $toBase);
$number = bcdiv($number, $toBase, 0);
```

So it will be even better.